src/app/components/universal/universal-facilities-card/universal-facilities-card.component.html

```diff
@@ -53,8 +53,40 @@
         </div>
         <div class="row">
           <div class="container ms-2 mt-2">
-            <ng-container *ngFor="let curData of data | keyvalue">
-              <div>{{ curData?.key?.substring(1) }}: <b>{{ curData?.value }}</b></div>
+            <ng-container *ngIf="isFiber">
+              <ng-container *ngIf="facilityIndex === 0">
```

Comment:
Is this overly complex? We could probably make universal components for this. Maybe as a future feature, and let it though now?

Reply:
this is a proof of concept/dummy components to fill out our pseudo "atlas", I don't think this is worth worrying about.